charger/openevse/api.go

```diff
@@ -2087,9 +2087,12 @@ type GetStatusResponse struct {
 		// WiFi RSSI signal strength
 		Srssi *int `json:"srssi,omitempty"`
 
-		// The EVSE status
+		// The EVSE state
 		State *int `json:"state,omitempty"`
 
+		// The EVSE status
+		Status *string `json:"status,omitempty"`
```

Comment:
Wow, tricky...

Reply:
This is undocumented in the OpenAPI spec, but tested it to work fine.